ddtrace/contrib/pytest/plugin.py

```diff
@@ -86,8 +155,28 @@ def pytest_configure(config):
         _CIVisibility.enable(config=ddtrace.config.pytest)
 
 
+def pytest_sessionstart(session):
+    if _CIVisibility.enabled:
+        test_session_span = _CIVisibility._instance.tracer.trace(
+            "pytest.test_session",
+            service=_CIVisibility._instance._service,
+            span_type=SpanTypes.TEST,
```

Comment:
is this the `'type'` tag in the span? This should then be `'test_session_end'`

Reply:
Same as above